public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +57,142 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Examples
 
-For these examples we can use the chapters in our
-[Get Started](/doc/get-started) section, up to
-[Add Files](/doc/get-started/add-files).
+For these examples we can use the [Get Started](/doc/get-started) repository.
 
 <details>
 
-### Click and expand to setup example
+### Click and expand to setup the examples project
 
-Start by cloning our example repo if you don't already have it. Then move into
-the repo and checkout the
-[3-add-file](https://github.com/iterative/example-get-started/releases/tag/3-add-file)
-tag, corresponding to the [Add Files](/doc/get-started/add-files) _Get Started_
-chapter:
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 $ cd example-get-started
-$ git checkout 3-add-file
 ```
 
-Download the precomputed data using:
+Download data using:
 
 ```dvc
-$ dvc pull
+$ dvc fetch -T
 Preparing to download data from 'https://remote.dvc.org/get-started'
 ...
 ```
 
+The `-T` flag passed to `dvc fetch` makes sure we have all the data files
+related to all existing tags in the repo. You take a look at the
+[available tags](https://github.com/iterative/example-get-started/tags) of our
+example repo.
```

Comment:
`You may see the available tags of our example repo [here](https://github.com/iterative/example-get-started/tags).`